config/longrun_configs/gpu_dyamond_target.yml

```diff
@@ -0,0 +1,15 @@
+anim: false
+atmos_config_file: "config/longrun_configs/longrun_aquaplanet_dyamond.yml"
+dt_cpl: 50
+dt_save_state_to_disk: "0.5days"
+dt_save_to_sol: "0.5days"
+energy_check: false
+job_id: "gpu_dyamond_target"
+land_albedo_type: "map_temporal"
+mode_name: "amip"
+mono_surface: false
+monthly_checkpoint: false
+run_name: "gpu_dyamond_target"
+start_date: "19790301"
+t_end: "1days"
```

Comment:
If it's a long run, could we run it for longer (e.g. 50 days) or does the simulation crash? 👀